qa/tasks/cephfs/kernel_mount.py

```diff
@@ -340,7 +340,7 @@ def _global_inst(self):
         if self.inst is not None:
             return self.inst
 
-        client_gid = "client%d" % self.get_global_id()
+        client_gid = "client%d" % int(self.get_global_id())
```

Comment:
if `get_global_id()` takes care of pre/post white spaces; string can be constructed directly 

```
client_gid = f"client{self.get_global_id()}"
```

Reply:
That is a valid suggestion, however, what made me do it the way it's done in this change is to explicitly typecast for catching the (type) issue. I'm fine with your suggestion too, btw.

Reply:
makes sense, maybe you can comment in the code or mention this in the commit message to make it self-explanatory